src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Mashine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator почав працювати.");
+    }
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator закінчив працювати.");

```

Comment:
The message in `stopWork()` is not in English. The task checklist requires all messages to be in English. Please update this message to be informative and in English.